src/d/actor/d_a_ks.cpp

```diff
@@ -5,125 +5,1613 @@
 
 #include "d/actor/d_a_ks.h"
 #include "d/d_procname.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_s_play.h"
+#include "f_op/f_op_camera.h"
+#include "d/d_snap.h"
+#include "d/actor/d_a_player_main.h"
+#include "d/actor/d_a_gm.h"
+#include "d/res/res_ks.h"
+
+#include "weak_bss_936_to_1036.h" // IWYU pragma: keep
+#include "weak_data_1811.h" // IWYU pragma: keep
+
+int KS_ALL_COUNT;
+int KUTTUKU_ALL_COUNT;
+BOOL HEAVY_IN;
+int GORON_COUNT;
```

Comment:
You can match the visibility of these objects by adding the static keyword to make them local. (Grey `l` in objdiff: local, green `g` in objdiff: global.)
```suggestion
static int KS_ALL_COUNT = 0;
static int KUTTUKU_ALL_COUNT = 0;
static BOOL HEAVY_IN = FALSE;
static int GORON_COUNT = 0;
```
Also, variables get initialized to 0 if no value is given, but specifying anyway it can be a bit more explicit and readable.